neural_network/activation_functions/binary_step.py

```diff
@@ -0,0 +1,37 @@
+"""
+This script demonstrates the implementation of the Binary Step function.
+
+It's an activation function in which if the input to the
+activation function is greater than a threshold, the neuron is
+activated, else it is deactivated
+
+It's a simple activation function which is mentioned in this wikipedia article:
+https://en.wikipedia.org/wiki/Activation_function
+"""
+
+
+import numpy as np
+
+
+def binary_step(vector: np.ndarray) -> np.ndarray:
+    """
+    Implements the binary step function
+
+    Parameters:
+        vector (ndarray): A vector that consitis of numeric values
+
+    Returns:
+        vector (ndarray): A vector that consitis of values 0 or 1
+
+    >>> vector = np.array([-1.2, 0, 2, 1.45, -3.7, 0.3])
+    >>> binary_step(vector)
+    array([0, 0, 1, 1, 0, 1])
+    """
+
+    return np.where(vector > 0, 1, 0)
```

Comment:
```suggestion
    >>> vector = np.array([-1.2, 0, 2, 1.45, -3.7, 0.3])
    >>> binary_step(vector)
    array([0, 1, 1, 1, 0, 1])
    """

    return np.where(vector >= 0, 1, 0)
```
Wikipedia states that the function should return 1 for an input of 0

Reply:
Thanks. I have resolved all comments.